src/us/kbase/assemblyhomology/core/Token.java

```diff
@@ -0,0 +1,61 @@
+package us.kbase.assemblyhomology.core;
```

Comment:
What will this token class be used for?

Reply:
It'll wrap the token string provided in the HTTP API by the caller, which will be passed into a filter (next PR) to filter out forbidden data.